coral-schema/src/main/java/com/linkedin/coral/schema/avro/ViewToAvroSchemaConverter.java

```diff
@@ -39,6 +40,7 @@
    */
   private ViewToAvroSchemaConverter(HiveMetastoreClient hiveMetastoreClient) {
     this.hiveToRelConverter = new HiveToRelConverter(hiveMetastoreClient);
+    this.relToAvroSchemaConverter = new RelToAvroSchemaConverter(hiveMetastoreClient);
```

Comment:
Could you verify if RelToAvroSchemaConverter is stateless and there is no side effect if creation is moved to constructor?

Reply:
It is, it only has an instance field which is a `HiveMetastoreClient`.

Reply:
Thanks @rzhang10 for verifying this.